code/mob/living/critter/bee.dm

```diff
@@ -262,12 +262,10 @@
 		var/obj/item/reagent_containers/food/snacks/pizza/floor_pizza = locate() in honeyTurf
 		var/obj/item/reagent_containers/food/snacks/ingredient/honey/honey
```

Comment:
please fix this pathing being wrong while your'e here, it should end at snacks

Reply:
Absolutely!